spec/helpstation/processors/by_key_fetcher_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Helpstation::Processors::ByKeyFetcher do
+  subject { processor.call request }
+
+  let(:processor) {
+    described_class.build(input_key, output_key) do
+      output
+    end
+  }
+
+  let(:input_key) { :my_input }
+  let(:output_key) { :my_output }
+  let(:output) { "putting out" }
+
+  let(:request)   { Substation::Request.new('my_request', env, input) }
+
+  context 'when input_key is present' do
+    let(:input) { { my_input: 'test' } }
+
+    it 'returns correct output' do
+      subject
+
+      expect(subject.class).to eq(Substation::Response::Success)
```

Comment:
`should be_a(Substation::Response::Success)`

or

`expect(subject).to be_a(Substation::Response::Success)`